docroot/modules/custom/va_gov_content_types/src/Traits/GetOriginalTrait.php

```
@@ -74,6 +82,11 @@ public function getOriginalField(string $fieldName): FieldItemListInterface {
   public function didChangeField(string $fieldName): bool {
     $originalField = $this->getOriginalField($fieldName);
     $currentField = $this->get($fieldName);
+    if (!$originalField) {
+      // Consider a new save (no original exists) to not be a change of value.
+      return FALSE;
```

I am torn on this a bit.  First commit, I was of the mind that going from no value to a value should be considered a change.  Second commit (Nate's patch) I aligned it with it not being a change.   They both seem equally misleading.

---

I'm not sure that it makes sense to call `didChangeField()` on something without an original version. I think it should probably throw an exception.

---

I can get behind that idea but it moves the defensiveness required higher up the chain we need to make less assumptions.